pkgs/applications/networking/instant-messengers/webex/default.nix

```diff
@@ -0,0 +1,95 @@
+{ lib, stdenv, fetchurl, patchelf, makeDesktopItem
+, alsa-lib, at-spi2-atk, at-spi2-core, atk, cups, dbus, expat, fontconfig
+, freetype, glib, harfbuzz, libdrm, libgcrypt, libglvnd, libkrb5, libpulseaudio
+, libsecret, udev, libxcb, libxkbcommon, lshw, mesa, nspr, nss, pango, zlib
+, libX11, libXcomposite, libXcursor, libXdamage, libXext , libXfixes, libXi
+, libXrandr, libXrender, libXtst, libxshmfence, xcbutil , xcbutilimage
+, xcbutilkeysyms, xcbutilrenderutil, xcbutilwm, p7zip, wayland
+, enableWayland ? false
+}:
+
+let
+  inherit (lib) optional;
+  name = "Webex";
+  binaryName = "CiscoCollabHost";
+  versionSpec = builtins.fromJSON (builtins.readFile ./version.json);
+
+in stdenv.mkDerivation rec {
+  pname = "webex";
+  version = versionSpec.version;
+
+  src = fetchurl {
+    url = versionSpec.packageLocation;
+    sha256 = versionSpec.checksum;
+  };
+
+  nativeBuildInputs = [ patchelf ];
+
+  buildInputs = [ alsa-lib at-spi2-atk at-spi2-core atk cups dbus expat
+                  fontconfig freetype glib harfbuzz lshw mesa nspr nss pango
+                  zlib libdrm libgcrypt libglvnd libkrb5 libpulseaudio libsecret
+                  udev libxcb libxkbcommon libX11 libXcomposite libXcursor
+                  libXdamage libXext libXfixes libXi libXrandr libXrender
+                  libXtst libxshmfence xcbutil xcbutilimage xcbutilkeysyms
+                  xcbutilrenderutil xcbutilwm p7zip]
+        ++ optional enableWayland wayland;
+
+  libPath = lib.makeLibraryPath buildInputs
+          + ":$out/opt/Webex/lib"
+          + ":$out/opt/Webex/bin";
+
+  unpackPhase = ''
+    7z x $src
+    mv Webex_ubuntu/opt .
+  '';
+
+  dontPatchELF = true;
+  dontStrip    = true;
```

Comment:
Why not?

Reply:
Might be leftovers from testing. Thanks for pointing it out. I'll see if it's actually necessary or not.